problems/countThrees.js

```diff
@@ -1,20 +1,27 @@
 /**
-* Counts the number of threes in a given number.
-* @param {number} n - The input number.
-* @returns {number} - Returns how many threes are in the input.
-*
-* ex: countThrees(300)
-* returns: 1
-*
-* ex: countThrees(3203)
-* returns: 2
-*
-* ex: countThrees(47)
-* returns: 0
-*/
+ * Counts the number of threes in a given number.
+ * @param {number} n - The input number.
+ * @returns {number} - Returns how many threes are in the input.
+ *
+ * ex: countThrees(300)
+ * returns: 1
+ *
+ * ex: countThrees(3203)
+ * returns: 2
+ *
+ * ex: countThrees(47)
+ * returns: 0
+ */
 
-function countThrees() {
+const countThrees = (n) => {
+  let output = 0;
+  let newString = n.toString();
+  for (let i = 0; i < newString.length; i++) {
```

Comment:
Try using for of loops when iterating through a string and not needing the index.